app/scripts/controllers/permissions/permissionsLog.js

```diff
@@ -25,7 +23,7 @@ export default class PermissionsLogController {
   /**
    * Get the activity log.
    *
-   * @returns {Array<Object>} - The activity log.
```

Comment:
Nit: we've been standardizing on using the dash to separate the description from the type elsewhere in the codebase. There are a few modules in the background that still lack the dash though.

Reply:
Hm, I want to push back on that. The reason I removed the dash was because it showed up in the description generated by VS Code. It doesn't get removed for `@returns` as it does for `@param`. Indeed, if we look at the JSDoc documentation, it does not include a dash for `@returns` in any of its examples:
- https://jsdoc.app/tags-returns.html
- Compare: https://jsdoc.app/tags-param.html

Reply:
Oh! Fair enough. Good find! I added the dash because I saw it was documented for `@param`, and I guess I just assumed the same was true of `@return`.

Reply:
We have had the exact same evolution of opinion on this matter 😄